src/api/app/controllers/webui/status_messages_controller.rb

```diff
@@ -71,6 +71,14 @@ def preview
 
   private
 
+  def require_staff
+    return if User.session.blank?
```

Comment:
With this line, we prevent from crashing for not logged in users for the `index`, `new` and `edit` actions. But we start permitting them.

```suggestion
```
Removing that line will make not logged in users will be shown the "Require staff privileges" message, at least. There would be better options, I guess.

Reply:
Hm, I thought I removed that... Anyway, I'll fix it now

Reply:
You even fixed the linter previously, same line...